src/Api/todos.tsx

```diff
@@ -0,0 +1,21 @@
+import { NewTodo, Todo } from '../types/Todo';
+import { client } from '../utils/fetchclient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const deleteTodo = (id: number) => {
```

Comment:
Consider adding a comment above the `deleteTodo` function to explain its purpose.